### ISSUE TITLE
[refactoring/base_02] changing language does not change it in UI

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#4104]
# Bug Report

## Description

changing language does not change it in UI

## Steps to reproduce

1. open app
2. go to settings
3. open language
4. switch to any language, for example Russian

#### Expected behavior

The app is shown in russian (for those strings that are translated)

#### Actual behavior

Language is english

<img width="1344" alt="Screenshot 2021-11-12 at 10 21 28" src="https://user-images.githubusercontent.com/82375995/141426823-6a2d9fe0-515f-4871-b052-a9fa52445f24.png">

### Additional Information

- Status desktop version: `refactoring_base02` branch, commit `224cb70`
- Operating System: masOS Big Sur

**Comment by Khushboo-dev-cpp** [comment]
@anastasiyaig I am not able to reproduce this bug. Can you please check. Assigning it to you.

https://images.zenhubusercontent.com/261270348/25596433-7b16-4766-b8f3-e00c3af83e84/screen_recording_2021_11_16_at_4_12_30_pm.mov

**Comment by anastasiyaig** [comment]
@Khushboo-dev-cpp on the video provided u are changing picture, not a language.

in https://github.com/status-im/status-desktop/commit/629aeccd8c0dc1b279fda65f963d5a269d65af69 i am still not able to change a language:


https://user-images.githubusercontent.com/82375995/142149623-d43f1f16-4e2a-4fdc-8888-79add50db4ea.mov

**Comment by Khushboo-dev-cpp** [comment]
Hey @anastasiyaig Sorry I uploaded the wrong video, actually there should be popup and only after a restart the change of language should apply, I will check and let you know.

**Comment by Khushboo-dev-cpp** [comment]
fix https://github.com/status-im/status-desktop/pull/4132 merged into refactoring/base_02.

@anastasiyaig the reason for the popup not showing up is that it is only shown on linux, on Mac OS it is expected that the language changed is observed instantly.
